src/nccl_ofi_rdma.c

```diff
@@ -2315,11 +2316,20 @@ static inline int post_bounce_buffs_on_rail(nccl_net_ofi_rdma_ep_t *ep,
 			NCCL_OFI_WARN("Failed to allocate bounce req");
 			return -ENOMEM;
 		}
+
+		if (i + 1 < buffers_needed) {
```

Comment:
Yeah, you're right.  I took a slightly different approach, because a flag to `send_progress()` seems icky.  But it's also kind of silly to have `post_bounce_buffs_on_rail()` calling a dispatch function like `send_progress()` anyway.  So instead I decided to just bypass the dispatch function and instead just call `post_bounce_buffer()` directly, and add a `set_fi_more` flag to that call, which is much less icky.